pages/linux/lsusb.md

```diff
@@ -0,0 +1,23 @@
+# lsusb
+
+> Display info about connected USB devices.
+
+- List all the USB ports available:
+
+`lsusb`
+
+- Hierarchy view, with each device's supported speed:
```

Comment:
Perhaps rephrasing to  `List the USB hierarchy as a tree` sounds a little better.